### ISSUE TITLE
eslint config is not connected to CRA

### ISSUE THREAD

**Issue by danieltott** [drewclem/protege#213]
## Describe the bug

This project is built using Create React App, which includes by default [a pretty extensive setup for eslint](https://www.npmjs.com/package/eslint-config-react-app). 

This project _also_ has an eslintrc file. However, you need to tell CRA to use this by adding `EXTEND_ESLINT="true"` to .env, which I'm pretty sure isn't ever done. What this means is that the lint errors you see running eslint manually are very different from the lint errors you see running `develop` or `build`. This is both confusing and error-prone.

Additionally the current eslint rules in place have some issues.

I'd be happy to submit a PR but I wanted to touch base first to see what your thinking is. 

My suggestion would be: 
- Remove the current eslintrc file entirely and uninstall those plugins
- Turn on `EXTEND_ESLINT` by default (this would probably involve adding an example .env file for people to copy
- Enable the `a11y` eslint rules included in `eslint-config-react-app`
- Update docs to reflect settings

I probably would also turn off `--fix` for `lint-staged` and instead get it to throw errors but that is probably a different issue.

Bottom line, the `eslint` command running in command line or lint-staged is running a completely different config than when run during `develop` or `build`. This is bound to introduce errors and/or rampant `/* eslint-disable */` usage.

The other option would be to turn `EXTEND_ESLINT` on by default and fix up the current eslint config. 

## To Reproduce

1. `eslint` in command line and see linting errors
2. run `npm run build` and see different linting errors

## Expected behavior

eslint has a deterministic result

### Additional context

- [Extending the ESLint config](https://create-react-app.dev/docs/setting-up-your-editor/#experimental-extending-the-eslint-config) in the CRA docs
- [eslint-config-react-app](https://www.npmjs.com/package/eslint-config-react-app)
-

**Comment by sethburtonhall** [comment]
Definitely a good Idea. I may jump into this to level up my eslint knowledge, I'll see.

**Comment by sethburtonhall** [comment]
Closing because issue was resolved incrementally by @nickytonline, @drewclem & @pickleat
